src/appsync.ts

```diff
@@ -295,20 +311,267 @@ export class AppsyncResolver<
     });
   }
 
-  private getResolvableFields(api: appsync.GraphqlApi) {
-    const resolverCount = countResolvers(this.decl);
+  private synthesizeFunctions(
```

Comment:
FYI - I moved this out of the class as a module-scoped function

Reply:
Which PR?

Reply:
#275 I imagine, will review once I stop the self mutation loop